spesifikasjoner/afpant/afpant-intensjon/afpant-intensjon.md

```diff
@@ -0,0 +1,21 @@
+# Intensjon om tinglysingsmetode
+## Introduksjon
+Tinglysing av hjemmelsovergang og kjøpers pantedokument kan utføres enten elektronisk eller på papir.
+
+Denne meldingstypen skal brukes for å koordinere valgt metode mellom kjøpers bank og megler.
+
+Signering av skjøtet og signering av pantedokumentet skjer på ulike tidspunkt i flyten, noe som kan skape utfordringer ved ulik intensjon mellom bank/megler.
+
+## Forutsetninger
+* Hver part sender sin intensjon
+* Megler tilgjengeliggjør sin intensjon i (megler)systemet for alle banker som måtte ønske å spørre på intensjon
```

Comment:
Omformulere? Det er ingen funksjon i løsningen som tilgjengeliggjør intensjonen for banker.

Reply:
Tror tilgjengeliggjør sin intensjon er ment som at alle kan spørre om intensjonen. Det er hverfall slik jeg tolker det, dvs at den er tilgjengelig for spørring.

Reply:
Så nettopp mailen fra Monica @ Ambita vedr dette, hun skriver:

Foil 3: «Megler tilgjengelig-gjør sin intensjon i systemet for alle banker som måtte ønske å spørre på intensjon» ønskes endret til «Megler tar stilling til sin intensjon i eget fagsystem»


Det er da en presisering for at ingen skal misforstå at de må forespørre intensjonen til megleren (og svaret kommer da automatisk, pga den er "tilgjengeliggjort" i meglersystemet)